sip-core/src/main/java/de/ikor/sip/foundation/core/trace/SIPTraceTypeEnum.java

```diff
@@ -0,0 +1,12 @@
+package de.ikor.sip.foundation.core.trace;
+
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor
+public enum SIPTraceTypeEnum {
+  BOTH("0"),
```

Comment:
Let's discuss if we need "both" option. 
Perhaps it would be clearer if the config looks like:
`sip.core.tracing.traceType: LOG, MEMORY
`

Reply:
Let's do this in the future if we get more tracing types.